make/source.mk

```diff
@@ -237,19 +253,35 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             sensors/gyroanalyse.c \
             $(CMSIS_SRC) \
             $(DEVICE_STDPERIPH_SRC) \
-            drivers/light_ws2811strip.c \
-            io/displayport_max7456.c \
-            io/osd.c \
-            io/osd_slave.c
 
 SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
+            blackbox/blackbox.c \
+            blackbox/blackbox_io.c \
```

Comment:
I don't think we should size optimise black box, since when logging is on it is speed critical.